os/drivers/lcd/mipi_lcd.c

```diff
@@ -169,12 +174,35 @@ static int lcd_putrun(FAR struct lcd_dev_s *dev, fb_coord_t row, fb_coord_t col,
 static int lcd_putarea(FAR struct lcd_dev_s *dev, fb_coord_t row_start, fb_coord_t row_end, fb_coord_t col_start, fb_coord_t col_end, FAR const uint8_t *buffer, fb_coord_t stride)
 {
 	FAR struct mipi_lcd_dev_s *priv = (FAR struct mipi_lcd_dev_s *)dev;
+
+	uint8_t *final_buffer = buffer;
+#ifdef CONFIG_LCD_ST7701SN
```

Comment:
Currently this supports landscape only. But we can do all of orientations in SW. Let's add them as well later.
```
#ifdef CONFIG_LCD_LANDSCAPE
  code
#elif defined(CONFIG_LCD_PORTRAIT)
  code
#elif defined(CONFIG_LCD_RPORTRAIT)
....
```

Reply:
Need to check here